UnityProject/Assets/Scripts/HealthV2/Living/LivingHealthMasterBase.cs

```diff
@@ -755,11 +756,60 @@ public void Extinguish()
 			healthStateController.SetFireStacks(0);
 		}
 
-		public virtual string GetExamineText()
+		#region Examine
+
+		public string GetExamineText()
 		{
-			return "Weeee";
+			if (this is PlayerHealthV2)
+			{
+				// Let ExaminablePlayer take care of this.
+				return default;
+			}
+
+			// Assume animal
+			var healthString = new StringBuilder("It is ");
+
+			if (IsDead)
+			{
+				healthString.Append("limp and unresponsive; there are no signs of life");
+
+				healthString.Append("...");
+
+				return healthString.ToString();
+			}
+
+			healthString.Append($"{ConsciousState.ToString().ToLower().Replace("_", " ")} and ");
+
+			var healthFraction = OverallHealth / maxHealth;
+			if (healthFraction < 0.2f)
+			{
+				healthString.Append("heavily wounded.");
+			}
+			else if (healthFraction < 0.6f)
+			{
+				healthString.Append("wounded.");
+			}
+			else
+			{
+				healthString.Append("in good shape.");
+			}
+
+			if (respiratorySystem != null && respiratorySystem.IsSuffocating)
+			{
+				healthString.Append("having trouble breathing!");
+			}
+
+			// On fire?
+			if (FireStacks > 0)
+			{
+				healthString.Append("And is on fire!");
```

Comment:
```suggestion
				healthString.Append("and is on fire!");
```

Reply:
humm, but the last added string has punctuation at the end, so it would be And

Reply:
Which last string? You mean the having trouble breathing? It is overwritten if on fire, as I consider that to be of more importance.

Reply:
Sorry...